### PR TITLE
Linux Mint Support

### PR DESCRIPTION
**Pull request by luckyraul**

### CODE REVIEW

File: tasks/install_shell.sh [hunk 1]
```diff
@@ -483,6 +483,26 @@ case $platform in
...
+  "Linuxmint")
+    info "Mint platform! Lets get you a DEB..."
+    case $major_version in      
+      "3")  deb_codename="stretch";;
+    esac
+    filetype="deb"
+    filename="${collection}-release-${deb_codename}.deb"
+    download_url="http://apt.puppetlabs.com/${filename}"
+    ;;
+  "LinuxMint")
```
Comment (ciprianbadescu) [inline tasks/install_shell.sh]: can you compact both cases in only one with "Linuxmint"|"LinuxMint" as pattern?

Comment (luckyraul) [reply tasks/install_shell.sh]: done

File: tasks/install_shell.sh [hunk 2]
```diff
@@ -483,6 +483,18 @@ case $platform in
     filename="${collection}-release-${deb_codename}.deb"
     download_url="${apt_source}/${filename}"
     ;;
+  "Linuxmint"|"LinuxMint")
+    info "Mint platform! Lets get you a DEB..."
+    case $major_version in
+      "3")  deb_codename="stretch";;
```
Comment (donoghuc) [inline tasks/install_shell.sh]: I do not have a way to test this, is "3" what the $major_version will resolve to? What version of the `facts` module are you using?

File: tasks/install_shell.sh [hunk 3]
```diff
@@ -483,6 +483,18 @@ case $platform in
...
+      "19") deb_codename="bionic";;
+      "18") deb_codename="xenial";;
+      "17") deb_codename="trusty";;
+    esac
+    filetype="deb"
+    filename="${collection}-release-${deb_codename}.deb"
+    download_url="http://apt.puppetlabs.com/${filename}"
```
Comment (nicklewis) [inline tasks/install_shell.sh]: This should use `${apt_source}/${filename}` as above.

File: tasks/install_shell.sh (hunk 2 above)
Comment (luckyraul) [reply tasks/install_shell.sh]: 3 is LDME
18, 19 is LinuxMint

**Pull request merged**